components/gateway/before-commit.sh

```diff
@@ -22,10 +22,9 @@ if [ "$1" == "$CI_FLAG" ]; then
 	buildEnv="env CGO_ENABLED=0 GOOS=linux GOARCH=amd64"
 fi
 
-${buildEnv} go build -o bin/app ./cmd/
-
+${buildEnv} go build -o gateway ./cmd/main.go
```

Comment:
Currently, this will fail unless you run explicitly `dep ensure -vendor-only`. Is this desired behavior?

Reply:
Added dep ensure at the beginning of the file